index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>landing page</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="js/index.js"></script>
+  </head>
+
+  <body>
+    <header id="header">
```

Comment:
. Kindly use class instead of ID, a class can be used to identify more than one HTML element while an ID is only used to identify one single element in our HTML.